cmds/deploy.go

```diff
@@ -160,7 +166,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 
 						tmpl.Parameters = append(tmpl.Parameters, tapi.Parameter{
 							Name:  "DOMAIN",
-							Value: domain,
+							Value: apidomain,
```

Comment:
I'm not quite sure. The DOMAIN template param is used as OAUTH_AUTHORIZE_URI and GOOGLE_OAUTH_REDIRECT_URI. So the AUTHORIZE_URI becomes: https://${DOMAIN}:${OAUTH_AUTHORIZE_PORT}/oauth/authorize. There it is correct to use the new apidomain param if set.. but the GOOGLE_OAUTH_REDIRECT_URI becomes: https://fabric8.${DOMAIN}. that might be wrong for the apidomain param. Maybe there needs to be two template params. One for the auth and one for the redirect.